WordPress/Classes/ViewRelated/Reader/Tab Navigation/ReaderTabView.swift

```diff
@@ -132,38 +130,21 @@ extension ReaderTabView {
     }
 
     private func activateConstraints() {
-        if viewModel.isSidebarModeEnabled {
-            mainStackViewTopAnchor = mainStackView.topAnchor.constraint(equalTo: buttonContainer.bottomAnchor)
-            guard let mainStackViewTopAnchor else { return }
-            NSLayoutConstraint.activate([
-                navigationMenu.centerXAnchor.constraint(equalTo: buttonContainer.centerXAnchor),
-                navigationMenu.topAnchor.constraint(equalTo: buttonContainer.topAnchor, constant: 8.0),
-                navigationMenu.bottomAnchor.constraint(equalTo: buttonContainer.bottomAnchor, constant: -8.0),
-                buttonContainer.topAnchor.constraint(equalTo: safeAreaLayoutGuide.topAnchor),
-                buttonContainer.leadingAnchor.constraint(equalTo: safeAreaLayoutGuide.leadingAnchor),
-                buttonContainer.trailingAnchor.constraint(equalTo: safeAreaLayoutGuide.trailingAnchor),
-                mainStackViewTopAnchor,
-                mainStackView.trailingAnchor.constraint(equalTo: safeAreaLayoutGuide.trailingAnchor),
-                mainStackView.leadingAnchor.constraint(equalTo: safeAreaLayoutGuide.leadingAnchor),
-                mainStackView.bottomAnchor.constraint(equalTo: bottomAnchor),
-            ])
-        } else {
-            mainStackViewTopAnchor = mainStackView.topAnchor.constraint(equalTo: buttonContainer.bottomAnchor)
-            guard let mainStackViewTopAnchor else { return }
-            NSLayoutConstraint.activate([
-                navigationMenu.leadingAnchor.constraint(equalTo: buttonContainer.leadingAnchor, constant: 12.0),
-                navigationMenu.trailingAnchor.constraint(equalTo: buttonContainer.trailingAnchor, constant: -16.0),
-                navigationMenu.topAnchor.constraint(equalTo: buttonContainer.topAnchor, constant: 8.0),
-                navigationMenu.bottomAnchor.constraint(equalTo: buttonContainer.bottomAnchor, constant: -8.0),
-                buttonContainer.topAnchor.constraint(equalTo: safeTopAnchor),
-                buttonContainer.leadingAnchor.constraint(equalTo: safeLeadingAnchor),
-                buttonContainer.trailingAnchor.constraint(equalTo: safeTrailingAnchor),
-                mainStackViewTopAnchor,
-                mainStackView.trailingAnchor.constraint(equalTo: safeTrailingAnchor),
-                mainStackView.leadingAnchor.constraint(equalTo: safeLeadingAnchor),
-                mainStackView.bottomAnchor.constraint(equalTo: safeBottomAnchor),
-            ])
-        }
+        mainStackViewTopAnchor = mainStackView.topAnchor.constraint(equalTo: buttonContainer.bottomAnchor)
```

Comment:
This code used to change the "pill" layout, but we are no longer going too display it or use this `ReaderTabView` at all.